src/main/drivers/serial_softserial.c

```diff
@@ -136,10 +210,9 @@ static void serialTimerTxConfig(const timerHardware_t *timerHardwarePtr, uint8_t
         }
     } while (isTimerPeriodTooLarge(timerPeriod));
 
-    uint8_t mhz = SystemCoreClock / 1000000;
+    uint8_t mhz = baseClock / 1000000;
```

Comment:
BTW: It would be great to remove the mhz nonsense - just pass the value around in Hz ...